include/psp2/ctrl.h

```diff
@@ -82,14 +87,32 @@ typedef struct SceCtrlData {
 	unsigned char 	rx;
 	/** Right analogue stick, Y axis. */
 	unsigned char 	ry;
-	/** Reserved. */
-	uint8_t		reserved0[4];
+	/** Up button */
+	uint8_t		up;
+	/** Right button */
+	uint8_t		right;
+	/** Down button */
+	uint8_t		down;
+	/** Left button */
+	uint8_t		left;
 	/** Left trigger (L2) */
-	unsigned char	lt;
+	uint8_t		l2;
 	/** Right trigger (R2) */
-	unsigned char	rt;
+	uint8_t		r2;
+	/** Left button (L1) */
+	uint8_t		l1;
+	/** Right button (R1) */
```

Comment:
these comments should change to just `L1 button` or something.

Reply:
I named them in that way cause in modern Dualshocks, L1 is the equivalent of LB and L2 the equivalent of LT in x360 joystick (so Left Button/Left Trigger).